certbot/main.py

```diff
@@ -384,6 +387,76 @@ def _tos_cb(regr):
     return acc, acme
 
 
+def _delete_if_appropriate(config): # pylint: disable=too-many-locals,too-many-branches
+    """Does the user want to delete their now-revoked certs?"""
+    display = zope.component.getUtility(interfaces.IDisplay)
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+
+    if not (config.certname or config.cert_path):
+        raise errors.Error('At least one of --cert-path or --cert-name must be specified.')
+
+    if config.certname and config.cert_path:
+        # first, check if certname and cert_path imply the same certs
+        cert_name_implied_cert_path = cert_manager.lineage_for_certname(config, config.certname)
```

Comment:
Ah, I think the mismatch comes from a refactoring that happened a while ago. Both options don't seem great, but since 1 seems like it could really backfire if they've messed with the files in a way we don't predict during 1's  implementation, I'm going to go with 2 despite it being slower.